File: exercise07/AreaOfRectangle.java
package exercise07;

import java.util.Scanner;

public class AreaOfRectangle {
    public static void main(String[] args) {
        final float CONVERSION_FACTOR = 0.09290304f;
        Scanner scan = new Scanner(System.in);
        int length, width, areaFeet;
        float areaMeters;
        String output;

        System.out.print("What is the length of the room in feet? ");
        length = scan.nextInt();
        System.out.print("What is the width of the room in feet? ");
        width = scan.nextInt();

        scan.close();

        areaFeet = length * width;
        areaMeters = areaFeet * CONVERSION_FACTOR;
        //m^2 = f^2 * 0.09290304

        output = String.format("You entered dimensions of %d feet by %d feet." 
            + "\nThe area is \n%d square feet\n%.3f square meters.", 
            length, width, areaFeet, areaMeters);

        System.out.print(output);
    }
}

File: 2. Calculations/exercise07/python/area_of_rectangle.py
# Exercise 07 - Area of a Rectangle

# Get input from user
length = input('What is the length of the room in feet? ')
width = input('What is the width of the room in feet? ')

# Convert input from string to integer
length = int(length)
width = int(width)

# Perform Calculations
area_in_feet = length * width
# Conversion formula -> m^2 = f^2 * 0.09290304
area_in_meters = round((area_in_feet * 0.09290304), 3)


# Format output
output_text = f'You entered dimensions of {length} feet by {width} feet.' \
              f'The area is\n' \
              f'{area_in_feet} square feet\n' \
              f'{area_in_meters} square meters'

print(output_text)
